### PR TITLE
feat: v1

### PR DESCRIPTION
**Pull request by madlabman**
image

TODO:
- [ ] Add actions to lint and check bundle actuality.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "action-verify-bytecode",
+  "version": "1.0.0",
+  "description": "Verify deployed contracts' bytecode",
+  "main": "dist/index.js",
+  "repository": "git@github.com:lidofinance/action-verify-bytecode.git",
+  "author": "madlabman <10616301+madlabman@users.noreply.github.com>",
```
Comment (skhomuti) [inline package.json:7]: not sure if it really what you want to put this

File: index.ts [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        msgParts.push(chalk.grey("@"));
+        msgParts.push(chalk.blue(value.desc.address));
+        const msg = msgParts.join("");
+
+        if (value.isMatched) {
+            core.info(msg);
+            continue;
+        }
+
+        core.setFailed(chalk.red(msg));
```
Comment (skhomuti) [inline index.ts:68]: image
looks not so good

fail messages (seems to be) already red

**Review by skhomuti — CHANGES_REQUESTED** [review]

File: index.ts (hunk 2 above)
Comment (skhomuti) [reply index.ts:68]: also failed check doesn't explain why it failed and what can (or should) I do with this, it is just the contract name with red cross

Comment (madlabman) [reply index.ts:68]: There is no simple answer to why a verification process has failed and you should do research by yourself in any case.

Comment (madlabman) [reply index.ts]: Redudant ANSI sequences removed.

File: package.json (hunk 1 above)
Comment (madlabman) [reply package.json:7]: What's wrong with it?

File: index.ts (hunk 2 above)
Comment (skhomuti) [reply index.ts]: I think a message like 'Contract MyAwesomeContract.sol bytecode differs from deployed byte code' will be better
look at this from the developer who doesn't know what actions check their contract point of view

File: package.json (hunk 1 above)
Comment (skhomuti) [reply package.json:7]: fans of this tool can't find you to send spam :(

File: index.ts (hunk 2 above)
Comment (madlabman) [reply index.ts]: Good point, I've updated the message.

File: package.json (hunk 1 above)
Comment (madlabman) [reply package.json]: Ok, gave them a chance.

**Review by dgusakov — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**